nodes/rare/rare.go

```diff
@@ -38,7 +38,8 @@ func (n *rare) Init(cfg *types.NodeConfig, opts ...nodes.NodeOption) error {
 
 	// make ipv6 disabled on all rare node interfaces unconditionally
 	// as ipv6 will be handled by rare/freertr
-	cfg.Sysctls["net.ipv6.conf.all.disable_ipv6"] = "1"
+	// The setting 'net.ipv6.conf.all.disable_ipv6' 1 - interferes with IPv6 out-of-band management. Commenting it out for now as a workaround.
+	// cfg.Sysctls["net.ipv6.conf.all.disable_ipv6"] = "1" 
```

Comment:
is it something you want to have a look at @frederic-loui?

Reply:
Thanks @hellt for getting back to me. We already had a discussion with @Takalele and [PR](https://github.com/srl-labs/containerlab/pull/2252) is the current simple way to enable IPv6 OOB. Hence we can approve the PR.